packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -5216,7 +5230,9 @@ export default class Meeting extends StatelessWebexPlugin {
       // @ts-ignore
       this.webex.internal.newMetrics.submitClientEvent({
         name: 'client.media.tx.start',
-        payload: {mediaType: data.type},
+        payload: {
```

Comment:
aren't we supposed to add `shareInstanceId` for "client.media.tx.start" and "client.media.tx.stop" too? (but only for "share" mediaType